### ISSUE TITLE
Street signs/warnings keep showing after navigation was stopped

### ISSUE THREAD

**Issue by Aiurek** [osmandapp/Osmand#7383]
Lenovo Moto G4 plus, Android 7.0
OsmAnd+ 3.4.xx, before the play store release 2019-08-03
May style: OsmAnd

Although navigation was finished/aborted Osmand occasionally kept showing me e.g. stop sign warnings on the screen.
Recognized this issue a couple of times with different street signs/warnings.

![Bild3](https://user-images.githubusercontent.com/27865200/62704224-26262d00-b9eb-11e9-8f23-c455cd2c2844.jpg)

**Comment by sonora** [comment]
Yes, it's by design, kind of a hidden feature. Also facilitating to have audio warnings about this (if you had voice navigation active) for users deciding to drive around and receive warnings with OsmAnd just following in map view mode, without an active navigation.

To stop it, just restart the app.

**Comment by vshcherb** [comment]
This is quite odd cause the feature @sonora is talking should not be in Browse Map. We are refactoring these settings now and we expect this should be fixed in 4.0 release
